lib/Doctrine/Common/DataFixtures/Loader.php

```diff
@@ -374,20 +370,18 @@ private function orderFixturesByDependencies()
     /**
      * @psalm-param iterable<class-string> $dependenciesClasses
      */
-    private function validateDependencies(iterable $dependenciesClasses): bool
+    private function validateDependencies(array $dependenciesClasses): void
```

Comment:
Why?

Reply:
Because method argument is an array not an iterable and there is no need to return boolean result - just throw an exception when validation is failed.

Reply:
> Because method argument is an array not an iterable 

What are you talking about? An array is an iterable, so it is both things at once.

Reply:
I understand that array is iterable, but when `validateDependencies` method is actually calling it always accept an array because `$fixture->getDependencies()` always return an array. It just what I want to said.

Reply:
Ok, it's always an array… so what? `validateDependencies` does not need it to be an array, it's only using it in a foreach, all it needs it to be is iterable.

Reply:
Ok, I just left only changes I wanted to do, without BC breaks